lib/Onnxifi/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ add_library(onnxifi-glow-thread-pool
 target_link_libraries(onnxifi-glow-lib
                       PUBLIC
                         ExecutionEngine
+                        Graph
```

Comment:
Transitive here, onnxifi-glow depends on EE and EE has Graph

Reply:
Does that actually fail locally?

Reply:
It does.  If I understand cmake correctly (which is questionable!), EE has Graph as a PRIVATE library dependence, so onnxifi-glow-lib won't see it.  I think we don't have this problem with static linking because libEE has the symbols from Graph linked in (maybe?  I didn't check that).

Reply:
Hm, but that doesn't really make sense, because it's working on CI... I'd like to understand what's up, but I'm inclined to land this and move on :-/